packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -407,6 +436,10 @@ export interface ChartProps extends VictoryChartProps {
    * @deprecated Use PatternFly's pf-theme-dark CSS selector
    */
   themeVariant?: string;
+  /**
+   * Generate default pattern defs and populate patternScale
+   */
+  usePatternDefs?: boolean;
```

Comment:
nit.  Typically in patternfly-core we prepend out booleans with `is` or sometimes `has`

Reply:
Changed to `isPatternDefs`